src/pathogena/util.py

```diff
@@ -325,16 +407,18 @@ def upload_fastq(
         dirty_checksum (str): The dirty checksum of the file.
     """
     reads = Path(reads)
-    logging.debug(f"upload_fastq(): {sample_id=}, {sample_name=}, {reads=}")
+    logging.debug(f"upload_fastq(): {batch_pk=}, {sample_name=}, {reads=}")
     logging.info(f"Uploading {sample_name}")
     checksum = hash_file(reads)
-    upload_file(
-        sample_id,
+    upload_file_as_chunks(
```

Comment:
`/uploads/upload-chunk/` api does a chunk index control as below;

```
            # check if the chunk has already been uploaded
            if sample.chunks.filter(chunk_order=chunk_index).exists():
```

so i think we should be creating the Sample in the format of upload api just like admin-app just to be consistent.